src/status_im/navigation/view.cljs

```diff
@@ -55,24 +55,25 @@
   [screen-key]
   (reagent.core/reactify-component
    (fn []
-     (let [screen-details              (get (if js/goog.DEBUG
-                                              (get-screens)
-                                              screens)
-                                            (keyword screen-key))
-           qualified-screen-details    (get (if js/goog.DEBUG
-                                              (get-screens)
-                                              screens)
-                                            (keyword "screen" screen-key))
-           {:keys [component options]} (or qualified-screen-details screen-details)
+     (let [screen-details                   (get (if js/goog.DEBUG
+                                                   (get-screens)
+                                                   screens)
+                                                 (keyword screen-key)) ;; todo - remove this, add
+                                                                       ;; :screen/ to all screens
```

Comment:
It will be big task for this PR to add qualifier to remaining screens and updating their uses, will require another PR